### PR TITLE
Replaced Client by own implementation to make use of additional request properties

### PR DESCRIPTION
**Pull request by FelsyWaschbaer**

### CODE REVIEW

File: server/middelwares/StanfordCoreNLPClient.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/**
+ * Created by Daniel on 14.03.2017.
+ */
```
Comment (kordianbruck) [inline server/middelwares/StanfordCoreNLPClient.js]: Please do not include these kind of comments.

File: server/middelwares/StanfordCoreNLPClient.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const rq=require('request-promise');
+
+class StanfordCoreNLPClient {
+    constructor(host="http://localhost:9000",annotators="tokenize,ssplit,pos", additionalProps = {}) {
```
Comment (kordianbruck) [inline server/middelwares/StanfordCoreNLPClient.js]: You should use a json config file for host/port.

**Review by kordianbruck — CHANGES_REQUESTED** [review]

**Comment by simonzachau** [comment]
very good to be able to include the coref knowledge in the openie results!

File: server/middelwares/StanfordCoreNLPClient.js (hunk 1 above)
Comment (FelsyWaschbaer) [reply server/middelwares/StanfordCoreNLPClient.js]: okay sry, it was automatically created by Webstorm

Comment (kordianbruck) [reply server/middelwares/StanfordCoreNLPClient.js]: I know, you can turn that "feature" off :+1:

**Review by kordianbruck — APPROVED** [review]

**Pull request merged**